pages/common/ansible-playbook.md

```diff
@@ -1,6 +1,7 @@
 # ansible-playbook
 
 > Execute tasks defined in playbook on remote machines over SSH.
+> Homepage: <https://www.ansible.com/>.
```

Comment:
Wouldn't these subcommand pages benefit from a more specific link, similar to what you did with `mysqldump`? IMO the root URL should only be used for the main command.

Reply:
Oh, good point! I haven't actually used _ansible_, so I must have skipped it. I'll do that now.